From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Mon, 5 Aug 2024 13:34:04 +0000
Subject: cherry-pick: Fix small typos in docker/build.sh

 - Missing `{` in `$NAMESPACE}` (it was in the manual push
   instructions, so it only broke manual copy-paste)

 - Use [] in usage message to indicate that namespace is optional

 - Use lowercase and hyphens for example namespace (Docker images
   names are typically lowercase), and change the name
---
 docker/build.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docker/build.sh b/docker/build.sh
index c4ba8d128..6cbd189b1 100755
--- a/docker/build.sh
+++ b/docker/build.sh
@@ -2,9 +2,9 @@
 
 if [ "$#" -eq 0 ] || [ "$#" -gt 2 ]
 then
-	echo "Usage: $0 domjudge-version <namespace>"
+	echo "Usage: $0 domjudge-version [namespace]"
 	echo "	For example: $0 5.3.0"
-	echo "	or: $0 5.3.0 otherNamespace"
+	echo "	or: $0 5.3.0 some-namespace"
 	exit 1
 fi
 
@@ -78,7 +78,7 @@ section_end
 section_start "Push instructions"
 echo "All done. Image ${NAMESPACE}/domserver:${VERSION} and ${NAMESPACE}/judgehost:${VERSION} created"
 echo "If you are a DOMjudge maintainer with access to the domjudge organization on Docker Hub, you can now run the following command to push them to Docker Hub:"
-echo "$ docker push ${NAMESPACE}/domserver:${VERSION} && docker push ${NAMESPACE}/judgehost:${VERSION} && docker push $NAMESPACE}/default-judgehost-chroot:${VERSION}"
+echo "$ docker push ${NAMESPACE}/domserver:${VERSION} && docker push ${NAMESPACE}/judgehost:${VERSION} && docker push ${NAMESPACE}/default-judgehost-chroot:${VERSION}"
 echo "If this is the latest release, also run the following command:"
 echo "$ docker tag ${NAMESPACE}/domserver:${VERSION} ${NAMESPACE}/domserver:latest && \
 docker tag ${NAMESPACE}/judgehost:${VERSION} ${NAMESPACE}/judgehost:latest && \
-- 
2.20.1

